sidebars.ts

```diff
@@ -298,6 +298,13 @@ const sidebars: SidebarsConfig = {
             'operator-guide/disaster-recovery/velero-restore-platform',
           ],
         },
+        {
+          'type': 'category',
```

Comment:
Do not create a new section. Let's put it under installation BEFORE 'Uninstall KubeRocketCI'

Reply:
We can call it 'Extensions'